### DISCUSSION TITLE
Earning Chia by farming - What's the reward?

### DISCUSSION THREAD

**Discussion by mfoglio** [Chia-Network/chia-blockchain#2236] [Q&A]
Hello everyone, I am new to the crypto world and I would like to try farming Chia. Out of curiosity, I created a plot of 100Gb. I know, it is small.
I can see from the GUI that my `Estimated time to win` is 3 years. What does it mean? Does it mean that, as of now, if I keep farming like this, I am likely to earn 1 chia in 3 years? And even more importantly, will I earn partial chia before 3 years or chances are that I will farm for 3 years for no reward?
I know that 100 Gb is not much space. I am just trying to understand how theory works. I am completely new to cryptos and chia: anything will help me understand better.
Thank you

**Comment by dorkmo** [comment]
chia awards are given out 2 at a time, so you'd get 2 chia every 3 years. but thats based on the current size of the total storage, so that timeframe will grow faster than the 3 year estimate. if you only want to use a small amout of space its suggested to join a pool. that way you'll only get a fraction of that 2chia on a more regular basis. pools are not 100% ready yet so you might have to wait a few weeks to join one.

**Reply by mfoglio** [reply]
Fantastic. So are there pool out yet? If yes, how can I find any?

**Reply by dorkmo** [reply]
ive seen a few pools, but i dont think theyre set up correctly yet bc they havent released the standard procedure for pools yet.

**Reply by mfoglio** [reply]
Thanks. If I may, where did you see them?

**Reply by dorkmo** [reply]
they list them here https://www.chiaexplorer.com/top-pools but im not sure if its multiple people or how to join. hpool.com is also a large pool, but im not sure if they have it set up bc everything is going into one wallet
